crates/relayer-types/src/core/ics02_client/height.rs

```diff
@@ -32,6 +32,11 @@ impl Height {
         })
     }
 
+    /// Yields the Height as a string.
+    pub fn as_str(&self) -> String {
+        format!("{}-{}", self.revision_number, self.revision_height)
+    }
+
```

Comment:
This method is unnecessary, instead you can call `.to_string()` on `Height` to get the same result.

This `to_string(&self) -> String` is method provided by the [`ToString`](https://doc.rust-lang.org/stable/std/string/trait.ToString.html) trait, which is automatically implemented for all types that implement the `Display` trait (which is the case for `Height`, see lines 148-152 below). See the linked doc for more info.

Reply:
Cool, I removed the method and used `to_string()`.